tests/sqlAst/data/sql-ast-narrowing.php

```diff
@@ -7,6 +7,61 @@
 
 class Foo
 {
+    public function whereIsNotNull(PDO $pdo): void
```

Comment:
please add tests which show that a WHERE from a subselect does not affect the outer selects select-columns